src/modules/template-details/TemplateDetails.tsx

```diff
@@ -100,28 +91,41 @@ const TemplateDetails = ({ match }: TemplateDetails.Props) => {
               <h3 className={csx.header}>
                 <span>Tech stack</span>
               </h3>
-              <ul className={`${csx.basicList} ${csx.white}`}>{mapList(template.technologies)}</ul>
+              <ul className={`${csx.basicList} ${csx.white} ${csx.technologies}`}>
+                {template.technologies.map(({ id, name }) => (
+                  <li key={id}>{name}</li>
+                ))}
+              </ul>
             </section>
 
             <section className={csx.col}>
               <h3 className={csx.header}>
                 <span>Patterns</span>
               </h3>
-              <ul className={`${csx.basicList} ${csx.white}`}>{mapList(template.patterns)}</ul>
+              <ul className={`${csx.basicList} ${csx.white} ${csx.patterns}`}>
+                {template.patterns.map(({ id, name }) => (
+                  <li key={id}>{name}</li>
+                ))}
+              </ul>
             </section>
 
             <section className={csx.col}>
               <h3 className={csx.header}>
                 <span>Authors</span>
               </h3>
-              <ul className={csx.authors}>{mapImages(template.contributors)}</ul>
+              <ul className={csx.authors}>
+                {template.contributors !== null &&
+                  template.contributors.map(({ name, avatar }) => {
+                    return (
```

Comment:
Avoid using return keyword in render map fn.

Reply:
Why?

Reply:
Because our jsx syntax should be short and simple. In every file we using just arrow return syntax. If you have something harder to write - you should create dedicated fn and put this inside map. This case is just simple map and adding return keyword is not needed. Always when you can avoid boilerplate you should do that. There are some edge cases when you shouldn't but in this case you should